docs/get-started.md

```diff
@@ -1,170 +1,64 @@
 
 Run the following commands to push up the first version of your new
-project to your remote code repository:
+prototype to your remote code repository:
 
 ```shell
 git add .
 git commit -m 'Initial commit'
 git push -u origin master
 ```
-
-
-### 5. Start working on your project
-
-Read the generated README for details on how to work on your project and
-the tools you will need to have installed. See also:
-[Working on your project]
-
-
-### 6. Optional: Set up Continuous Integration
-
-We provide some workflows for [GitHub Actions] to help you set-up
-Continuous Integration (CI) for your project. This allows you to gain
-assurance over proposed changes to your project by doing things like
-running unit tests, visual regression tests on components, and building
-and testing your applications.
-
-The unit tests and builds should run with no extra set-up if you are
-using GitHub. (You will need translate them into another CI system if you
-wish to use something else.) The others will require some work to set
-them up.
-
-
-#### 6.1. Chromatic
-
-We provide a workflow for publishing your Storybook to the [Chromatic]
-service. This allows you to detect and review any visual changes made to
-your components and so avoid accidental regressions.
-
-1. Sign up and [log in to Chromatic]
-2. Add your project
-3. Take note of your 'project token'
-4. Create a new secret in GitHub called `CHROMATIC_PROJECT_TOKEN` with
-   the value set to the one provided to you by Chromatic.
-
-Once that is done you should be able to detect and review visual changes
-to your components.
-
-You should consider making these checks mandatory to prevent unauthorised
-changes being merged in to your `master` branch.
-
-
-### 7. Optional: Set up Continuous Deployment
-
-The included 'docs' application can be automatically deployed to
-[Netlify]. This allows you to introduce people to your project as well
-as any components and libraries that you publish whilst remaining 'on
-brand'.
-
-1. Sign up and [log in to Netlify]
-2. Create a new site by following the [Netlify documentation]
-   To avoid giving unnecessary access to Netlify, consider using the CLI
-   tool as follows:
-   ```shell
-   npm install netlify-cli -g
-   netlify login
-   netlify init --manual
-   ```
-   You can safely ignore the ssh key and webhook as we will be building
-   on GitHub Actions instead of Netlify.
-3. Create a new secret in GitHub called `NETLIFY_SITE_ID` by following
-   the information here:
-   https://docs.netlify.com/cli/get-started/#link-with-an-environment-variable
-4. Create a new secret in GitHub called `NETLIFY_AUTH_TOKEN` by following
-   the information here:
-   https://docs.netlify.com/cli/get-started/#obtain-a-token-in-the-netlify-ui
-
-Once this is in place your Netlify site should be updated whenever you
-push to your `master` branch.
-
-Once your website is up and running, you should consider linking to it
-from your README, your `package.json` and your GitHub repository.
```

Comment:
We should actually reinstate this section at some point.

For now though, I've not included the script for deploying in the prototype so it wouldn't actually work.

Reply:
Have reinstated this under apps/docs/src/common/pages/get-started/project.tsx in [PR #57](https://github.com/UKHomeOffice/hods-poc/pull/57)